src/components/side_panel/data_validation/dv_criterion_form/dv_input/dv_input.xml

```diff
@@ -1,20 +1,7 @@
 <templates>
   <t t-name="o-spreadsheet-DataValidationInput">
     <div class="o-dv-input position-relative w-100">
```

Comment:
Idk if it was a problem before or if is now because it's a composer, but the error icon should have some margin at the bottom and not be stuck to the border 
![image](https://github.com/user-attachments/assets/68d551f3-0044-4532-86df-d3fe7cd87a0c)